pkg/apis/config/testdata/feature-flags-invalid-enable-api-fields.yaml

```diff
@@ -18,4 +18,4 @@ metadata:
   name: feature-flags
   namespace: tekton-pipelines
 data:
-  enable-api-fields: "im-not-a-valid-feature-gate"
+  embedded-status: "im-not-a-valid-feature-gate"
```

Comment:
I think this change was made to the incorrect file.

Reply:
Yeah, that's a boo-boo. Fixing!